### DISCUSSION TITLE
AS4 multi-hop and EESSI Profile

### DISCUSSION THREAD

**Discussion by mvolkert** [phax/phase4#164] [General]
We would like to use phase4 as an Endpoint (not the intermediary) of an muli-hop message exchange.
But we saw that this is not implemented: https://github.com/phax/phase4/wiki/Known-Limitations.
How much does an Endpoint of an AS4 multi-hop need to know about that it is multi-hop ?
Might it already work or how much work would it be to add this feature ?

Do you know about the AS4 EESSI profile, that is supported by those two AS4 implementations ?:
http://holodeck-b2b.org/features/
https://ec.europa.eu/digital-building-blocks/code/projects/EDELIVERY/repos/eessi-as4.net/browse
We would like to use this profile, it is somewhat similar to the cef profile, but I am unsure how much work it would be to add it.

**Comment by phax** [comment]
@mvolkert  I haven't really dived into Multi Hop and its requirements, I am pretty sure that it can easily be added but currently I have neither time nor plans to implement it.

Regarding the EESSI profile - I also didn't look into this one particularly but I am also quite sure, that adding it should not be a big deal, but with the same limitations as above - its currently quite a stressful time for me - sorry.
